src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1282,11 +1540,41 @@ private function step_in_body() {
 				$this->run_adoption_agency_algorithm();
 				return true;
 
+			/*
+			 * > A start tag whose tag name is one of: "applet", "marquee", "object"
+			 */
+			case '+APPLET':
+			case '+MARQUEE':
+			case '+OBJECT':
+				$this->reconstruct_active_formatting_elements();
+				$this->insert_html_element( $this->state->current_token );
+				$this->state->active_formatting_elements->add_marker();
+				$this->state->frameset_ok = false;
+				return true;
+
+			/*
+			 * > A start tag whose tag name is "table"
+			 */
+			case '+TABLE':
+				if (
+					WP_HTML_Processor_State::QUIRKS_MODE !== $this->state->compat_mode &&
+					$this->state->stack_of_open_elements->has_p_in_button_scope()
+				) {
+					$this->close_a_p_element();
+				}
+
+				$this->insert_html_element( $this->state->current_token );
+				$this->state->frameset_ok    = false;
+				$this->state->insertion_mode = WP_HTML_Processor_State::INSERTION_MODE_IN_TABLE;
+				return true;
+
 			/*
 			 * > An end tag whose tag name is "br"
 			 * >   Parse error. Drop the attributes from the token, and act as described in the next
 			 * >   entry; i.e. act as if this was a "br" start tag token with no attributes, rather
 			 * >   than the end tag token that it actually is.
+			 *
+			 * Note that this is handled elsewhere, so it should not be possible to reach this code.
```

Comment:
Do we want to update the unsupported message? If it's handled elsewhere and we don't expect to hit this case the message seems to be outdated.

I do still see an unsupported error with `</br>`.

Reply:
that was an aspirational comment late at night.

as of 9cc67b0 it should only ever report as an opening BR, even in the Tag Processor